cmake/coverage.cmake

```diff
@@ -338,6 +338,15 @@ function(coverage_target_clean_intermediate_file)
 endfunction()
 
 
+set_property(GLOBAL PROPERTY global_coverage_test_targets)
```

Comment:
I'm not sure if there's a neater way of doing this.

Reply:
> I'm not sure if there's a neater way of doing this.

I think this is the least invasive approach.